packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -1073,8 +1140,13 @@ contract GenArt721CoreV3_Engine is
         if (_secondMarketRoyalty > ARTIST_MAX_SECONDARY_ROYALTY_PERCENTAGE) {
```

Comment:
Super minor nit/not a new change but do we want to use `_secondaryMarketRoyalty` instead of `_secondMarketRoyalty` for consistency?

Reply:
absolutely! updated in  [aaf33ff](https://github.com/ArtBlocks/artblocks-contracts/pull/1455/commits/aaf33ff2b1542cbfd5c7f7a1b789b4c7932c07ad)